docs/src/components/DocumentationLayout.tsx

```diff
@@ -76,11 +125,25 @@ const DocumentationLayout = ({ children, seo }: Props) => {
           <DefaultBanner content={content} />
         )}
       />
-      <Navigation />
+      <Navigation
+        isSidebarOpen={isSidebarOpen}
+        setIsSidebarOpen={setIsSidebarOpen}
```

Comment:
```suggest
onSidebarOpen={setIsSidebarOpen}
```

let's have a convention to name all callback props same style

Reply:
corrected